docker/Dockerfile

```diff
@@ -4,21 +4,39 @@ SHELL ["/bin/bash", "-c"]
 
 RUN apt-get update && \
     apt-get install -y \
-    can-utils \
-    git \
-    python3-catkin-tools \
-    python3-vcstool && \
+        can-utils \
+        git \
+        python3-catkin-tools \
+        python3-vcstool && \
+    mkdir -p /root/canopen_ws/src && \
+    cd /root/canopen_ws && \
+    catkin init && \
+    cd src && \
+    git clone -b melodic-devel https://github.com/smilerobotics/ros_canopen.git && \
+    rosdep install --from-paths . -i -y && \
+    source /opt/ros/noetic/setup.bash && \
+    catkin config --install && \
+    catkin build && \
+    catkin clean -bdL -y && \
     mkdir -p /root/catkin_ws/src && \
     cd /root/catkin_ws && \
-    catkin init && \
-    cd /root/catkin_ws/src && \
-    git clone https://github.com/smilerobotics/haruna.git && \
-    vcs import < haruna/.rosinstall && \
-    rosdep install --from-paths . -y && \
+    catkin init
```

Comment:
If we care about the image size, we should remove apt cache by 'rm -rf /var/lib/apt/lists/*', because L16 do apt install.
(But it increases the build time because we will do ti again in L31.

Do you have good idea?

Reply:
> (But it increases the build time because we will do ti again in L31.

That's exactly why I didn't do so.

> Do you have good idea?

If we install all the needed packages from haruna in `apt-get install` beginning from L6,
we may possibly not need `rosdep install` in L31.
(But we have to update the `apt-get install` lines, every time the haruna dependencies are changed.)